dsaas-services/f8-toggles-service.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+- hash: 43598e780bf29b9e330d48a445cce838f6523fe3
```

Comment:
Change this hash to 7f988ff0c74ccbfb0cd1e56d04f698d0ff55be4d to get the IMAGE_TAG fix

Reply:
right! let me do that right now